assets/scss/_fonts.scss

```diff
@@ -1 +1,2 @@
 $primary-font: Poppins;
+$secondary-font: Poppins;
```

Comment:
what's the difference between this two variables? I mean, primary and secondary?

Reply:
apologies was supposed to be Roboto